valio-master/Valio/ItemTableViewCell.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  ItemTableViewCell.swift
+//  Valio
+//
+//  Created by Sam Soffes on 6/6/14.
+//  Copyright (c) 2014 Sam Soffes. All rights reserved.
+//
+
+import UIKit
+import QuartzCore
+
+class ItemTableViewCell: UITableViewCell {
+	
+	// MARK: - Properties
+
+	lazy var titleLabel: UILabel = {
+		let label = UILabel()
+		label.setTranslatesAutoresizingMaskIntoConstraints(false)
+		label.font = UIFont(name: "Avenir", size: 12)
+		return label
+	}()
+	
+	lazy var timeLabel: UILabel = {
+		let label = UILabel()
+		label.setTranslatesAutoresizingMaskIntoConstraints(false)
+		label.font = UIFont(name: "Avenir-Light", size: 12)
+		label.textColor = UIColor(red: 0.631, green: 0.651, blue: 0.678, alpha: 1)
```

Comment:
A better idea would be to create a category on `UIColor` with all those colors named, as this is hard to keep track, code review, etc. With the category you can do more semantic color naming, like `textColor` or `cellTimeTextColor` or use a palette of the same colors and not think up new ones every time you add something.